src/Module/Friendica.php

```diff
@@ -125,6 +127,7 @@ protected function rawContent(array $request = [])
 				$data = ActivityPub\Transmitter::getProfile(0);
 				header('Access-Control-Allow-Origin: *');
 				header('Cache-Control: max-age=23200, stale-while-revalidate=23200');
+				header('Vary: Accept', false);
```

Comment:
Please move this statement to line 119 so that all the cases are covered by the `Vary` header.